0.3-tagged-hashes.ipynb

```diff
@@ -74,7 +88,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.5"
```

Comment:
I dont know if it matters but I think most (all?) notebooks are 3.6.5.

Reply:
oops. Didn't mean to change that. I'll revert